versions.cfg

```diff
@@ -25,9 +25,12 @@ ZopeUndo = 4.1
 coverage = 4.2
 zdaemon = 4.1.0
 transaction = 1.4.4
-Sphinx = 1.3.6
+Sphinx = 1.5.1
+unittest2 = 1.1.0
```

Comment:
it is used by robotsuite and http://dist.plone.org/versions/zopetoolkit-1-0-8-ztk-versions.cfg pins a very old 0.5.1